Orzqwq scraping issue

I am having an issue downloading image sets in the form of a manga. Unsure if this is an issue with how the website is structured or an issue on my end.

Tried various methods to download
https://orzqwq.com/manga/camonome-bukatsu-chokugo-no-rikujoubu-joshi-ni-sasowarete/

get-sauce https://orzqwq.com/manga/camonome-bukatsu-chokugo-no-rikujoubu-joshi-ni-sasowarete/japanese/?style=list

 Site:      https://orzqwq.com/
 Title:
 Type:      image
 Stream:

     [0]  -------------------
     Type:            image
     Quality:         unknown
     Size:            0 B
     # download with: get-sauce -s 0 ...

get-sauce -p 1-61 https://orzqwq.com/manga/camonome-bukatsu-chokugo-no-rikujoubu-joshi-ni-sasowarete/

 Site:      https://orzqwq.com/
 Title:     [Camonome] Bukatsu Chokugo no Rikujoubu Joshi ni Sasowarete
 Type:      image
 Stream:

     [0]  -------------------
     Type:            image
     Quality:         unknown
     Parts:           61
     Size:            0 B
     # download with: get-sauce -s 0 ...

Downloaded 61 blank images

get-sauce https://orzqwq.com/manga/camonome-bukatsu-chokugo-no-rikujoubu-joshi-ni-sasowarete/japanese/p/1/

 Site:      https://orzqwq.com/
 Title:     [Camonome] Bukatsu Chokugo no Rikujoubu Joshi ni Sasowarete
 Type:      image
 Stream:

     [0]  -------------------
     Type:            image
     Quality:         unknown
     Size:            0 B
     # download with: get-sauce -s 0 ...

get-sauce -p 1-61 https://orzqwq.com/manga/camonome-bukatsu-chokugo-no-rikujoubu-joshi-ni-sasowarete/japanese/p/1/
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/gan-of-culture/get-sauce/extractors/orzqwq.extractData({0xc00007e060, 0x60})
        C:/Users/XXXXX/go/pkg/mod/github.com/gan-of-culture/get-sauce@v1.2.37/extractors/orzqwq/orzqwq.go:92 +0x698
github.com/gan-of-culture/get-sauce/extractors/orzqwq.(*extractor).Extract(0x3c5ba0?, {0xc00007e060?, 0xc00007e068?})
        C:/Users/XXXXX/go/pkg/mod/github.com/gan-of-culture/get-sauce@v1.2.37/extractors/orzqwq/orzqwq.go:34 +0xc5
github.com/gan-of-culture/get-sauce/extractors.Extract({0xc00007e060, 0x60})
        C:/Users/XXXXX/go/pkg/mod/github.com/gan-of-culture/get-sauce@v1.2.37/extractors/extractors.go:127 +0xb6
main.download({0xc00007e060?, 0x60?})
        C:/Users/XXXXX/go/pkg/mod/github.com/gan-of-culture/get-sauce@v1.2.37/main.go:46 +0xb5
main.main()

get-sauce https://orzqwq.com/manga/camonome-bukatsu-chokugo-no-rikujoubu-joshi-ni-sasowarete/japanese/?style=list

 Site:      https://orzqwq.com/
 Title:     
 Type:      image
 Stream:   

     [0]  -------------------
     Type:            image
     Quality:         unknown
     Size:            0 B
     # download with: get-sauce -s 0 ...

I did notice that there was no direct image address unless I went to the images on the page/list and copied the address from the image. When that url was used in get-sauce it was downloaded. The images are in the format:

https://img2.orzqwq.com/manga_62910f394c745/4fff8e79f1f2db9b2e349f1b1f770920/001.jpg
https://img2.orzqwq.com/manga_62910f394c745/4fff8e79f1f2db9b2e349f1b1f770920/002.jpg
https://img2.orzqwq.com/manga_62910f394c745/4fff8e79f1f2db9b2e349f1b1f770920/003.jpg
and so on

Is there a different command that I should be using for this website?

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -38,5 +38,6 @@ changelog:
     exclude:
       - '^docs:'
       - '^test:'
+      - '^md:'
 
   

--- a/extractors/nhgroup/nhgroup_test.go
+++ b/extractors/nhgroup/nhgroup_test.go
@@ -122,7 +122,7 @@ func TestExtract(t *testing.T) {
 				Size:    424461512,
 			},
 		},
-		{
+		/*{
 			Name: "Single Episode hentaihaven.com",
 			Args: test.Args{
 				URL:     "https://hentaihaven.com/usamimi-bouken-tan-sekuhara-shinagara-sekai-o-sukue-episode-3/",
@@ -130,7 +130,7 @@ func TestExtract(t *testing.T) {
 				Quality: "1920x1080",
 				Size:    143517696,
 			},
-		},
+		},*/
 		{
 			Name: "Single Episode",
 			Args: test.Args{

--- a/extractors/orzqwq/orzqwq.go
+++ b/extractors/orzqwq/orzqwq.go
@@ -14,7 +14,7 @@ import (
 
 const site = "https://orzqwq.com/"
 
-var reImageURL = regexp.MustCompile(`image-\d+"\ssrc="([^"]+)`)
+var reImageURL = regexp.MustCompile(`image-\d+"\s.+data-src="([^"]+)`)
 
 type extractor struct{}
 

